airflow/providers/cncf/kubernetes/operators/job.py

```diff
@@ -135,6 +144,18 @@ def execute(self, context: Context):
         ti.xcom_push(key="job_name", value=self.job.metadata.name)
         ti.xcom_push(key="job_namespace", value=self.job.metadata.namespace)
 
+        if self.wait_job_until_complete:
```

Comment:
what happens if the job is suspended externally? are we going to wait indefinitely? Or can we have wait timeout?